### ISSUE TITLE
Searching DNB for existing ISBN returns empty result

### ISSUE THREAD

**Issue by workflowsguy** [citronalco/calibre-dnb#4]
When searching DNB from the command line for an ISBN that has associated book data,
it returns "No results found".

Steps to reproduce

Input:
"/Applications/calibre.app/Contents/MacOS/fetch-ebook-metadata" -i 9783641042738 -p DNB_DE

Output:
<calibre.utils.logging.ThreadSafeLog object at 0x116555650>
No results found

Expected:
Book data for title "Die Neuromancer-Trilogie : Roman / William Gibson" as is shown on https://portal.dnb.de/opac.htm?query=9783641042738&method=simpleSearch&cqlMode=true

Other ISBNs I found with the same issue:

9783833869907
9783709306406

Calibre version 3.48 on macOS 10.13.6

**Comment by citronalco** [comment]
I never tried it from command line - up to now. So maybe I'm doing something wrong, but I tried with all plugins I've had available, and got the same output as you did. With every single one of them. I'm also using Calibre 3.48.

Does it work for you with any other plug in? If yes: Which one?

**Comment by workflowsguy** [comment]
> Does it work for you with any other plug in? If yes: Which one?

I am not sure which other plugins you are referring to. My searches so far (both in Calibre and on the mobileread forum) have not turned up any other plugins that expressly provide DNB as a data source.

**Comment by citronalco** [comment]
What I meat is: I tried a few other plugins, ie. for searching in Amazon, or on Google, and none of them returned anything else than
```
<calibre.utils.logging.ThreadSafeLog object at 0x116555650>
No results found
```
Regardless whether I did a search for ISBN or title+author.

But I believe I found the bug in my plugin. Will try to fix it within the next days.

**Comment by citronalco** [comment]
Bug is supposed to be fixed.
Could you try the new release and give feedback? You'll have to download it from this repository's "Releases" page, so far I haven't implemented any update mechanism into the plugin itself.

**Comment by workflowsguy** [comment]
I did a quick test with several ISBNs of german book titles using the  `-p DNB_DE` option.
They all returned the book's metadata correctly.

Thank you for adressing this issue so quickly!

Just as a follow-up regarding your question about other plugins working:
Before you fixed the issue, the output from `fetch-ebook-metadata` for 9783833869907 was:

```
"/Applications/calibre.app/Contents/MacOS/fetch-ebook-metadata" -i 9783833869907 -p DNB_DE
<calibre.utils.logging.ThreadSafeLog object at 0x111373650>
No results found

"/Applications/calibre.app/Contents/MacOS/fetch-ebook-metadata" -i 9783833869907 -p Google
Title               : KINTSUGI - Scherben bringen Glück: Die Kunst, unsere Wunden zu heilen
Author(s)           : Frank, Pascal Akira
Publisher           : Gräfe Und Unzer
Tags                : Family & Relationships, General, Self-Help, Personal Growth
Languages           : deu
Published           : 2019-02-05T15:17:24.298296+00:00
Identifiers         : isbn:9783833869907, google:_YN-DwAAQBAJ
Comments            : O Schreck! Herr Takeshi lässt seine Lieblingstasse fallen. Herr Nakamura gerät in eine Lebenskrise. Was ist beiden gemein? Kintsugi! Die traditionelle japanische Handwerkskunst zeigt nicht nur, wie man zerbrochene Keramik kittet, sondern auch, wie man emotionale Wunden heilt: geduldig, achtsam und mit Mitgefühl. Das Besondere: Der Kintsugi-Prozess (jap. Kint = Gold; Sugi = verbinden) erfordert große Sorgfalt, doch am Ende werden die Bruchstellen veredelt und die Tasse erstrahlt mit ihren goldenen Adern kostbarer als zuvor. So wird er zu einer Metapher der wertvollen Heilung. Herr Takeshi trägt also seine geliebte Tasse zu einem Kintsugi-Meister. Herr Nakamura erlebt die Phasen des Kintsugi an Leib und Seele. Am Ende ist nicht nur die zerbrochene Tasse wieder heil, sondern auch das Leben von Herrn Nakamura. Seine Beziehungen sind tiefer, sein Leben ist reicher geworden. Eine zauberhafte Geschichte aus dem Herzen Japans. Mit hilfreichen Übungen und Reflexionen für den Leser.

```
(For this ISBN, the plugin for Amazon also returned "no results found")
